README.md

```diff
@@ -13,10 +13,11 @@ Note: General instructions for git / git workflow can be found on our group's OS
 
 #### II. How to add a team member to the People section
 
-1. Within the repo in RStudio, open the `content` > `people` folders
-2. Create a folder with the naming format of `new-person`
-3. To include a photo or headshot in the bio, upload a `.png` image to that person's folder named `featured.png`
-4. Copy `index.md` from another team member's folder and update with new information. Remember to change the team member tag to `current`, `affiliate`, or `past` as needed.
+1. Within the repo in RStudio, open the `content` > `authors` folders
+2. Create a folder with the naming format of `firstname-lastname`
+3. To include a photo or headshot in the bio, upload a `.png` image to that person's folder named `avatar.png`
+4. Copy `_index.md` from another team member's folder and update with new information. Remember to: update `Display name`, `Role/position`, and `Social/Academic Networking`; set `user_groups` to "Current Members", "Affiliates", or "Alumni"; and add a paragraph or two about the person's interests, role in the group, and past experiences at the end after the yml. 
+5. Run `blogdown::serve_site()` in the RStudio console and check social media links are correct on new person page
```

Comment:
Do we have instructions on pushing changes to the public website?

Reply:
I think currently anything that is merged into this repo changes [the public website](https://quizzical-perlman-731692.netlify.app/)?

Reply:
Could you add something to the top of this file that says something along the lines of "Pushing changes to the `master` branch automatically publishes the web site"? (I can't see how to suggest the change in this PR, otherwise I'd do it)

Reply:
@Chris-Schnaufer just did, see [the commit](https://github.com/az-digitalag/group-website/pull/16/commits/0a737e9b36652163f9751c836dd358b2341452ea) below.

Yeah, I think suggestions can be made on only the lines that already have changes, and a few lines before and after.